config/locales/en.yml

```diff
@@ -256,6 +256,7 @@ en:
         retain_notes: Your map notes and note comments, if any, will be retained but hidden from view.
         retain_changeset_discussions: Your changeset discussions, if any, will be retained.
         retain_email: Your email address will be retained.
+        recent_editing_html: "Due to your recent editing activity you'll be able to delete your account in %{time}"
```

Comment:
This doesn't read very well to me but I'm struggling to come up with a better phrasing, at least as a single sentence. The best I've come up with is:

> As you have edited recently your account cannot currently be deleted.
> Deletion will be possible in %{time}.

Reply:
How about:

Based on your recent mapping activities, deletion of your account is only possible in x days.

Reply:
Changed the message.

I started with the block expiry message to avoid introducing some grammar that might cause trouble during internationalization.